pkg/operators/v1alpha1/clusterserviceversion_types.go

```diff
@@ -204,7 +206,7 @@ func (w *WebhookDescription) GetValidatingWebhook(namespace string, namespaceSel
 				Name:      w.DomainName() + "-service",
 				Namespace: namespace,
 				Path:      w.WebhookPath,
-				Port:      &w.ContainerPort,
+				Port:      w.ContainerPort,
```

Comment:
Is this change intentional? It actually causes the failure on the test because of this. You will need to update the test case if that is the case.

Reply:
Yes - we'll want to know when this value is actually set.

Reply:
I suppose that we still get that information if it is not a ptr because the value will be 0.

Reply:
I meant the type change but look like you corrected on the test.